src/cqerl_client.erl

```diff
@@ -537,6 +546,11 @@ handle_info({ Transport, Socket, BinaryMsg }, sleep, State = #client_state{ sock
             end,
             {next_state, sleep, State#client_state{delayed=Delayed}, Duration1};
 
+        {ok, #cqerl_frame{opcode=?CQERL_OP_SUPPORTED}, _Payload, Delayed} ->
+            %% SUPPORTED is the message received from Cassandra for OPTIONS request,
+            %% which we use for heartbeats. We need to handle that for live state without any State modifications.
```

Comment:
Copy'n'paste artifact - this is not `live` state

Reply:
True, fixed.